src/wdio/services/TerraToolkitServeStaticService.js

```diff
@@ -0,0 +1,44 @@
+import serve from '../../../scripts/serve/serve-static';
+
+export default class TerraToolkitServeStaticService {
+  async onPrepare(config) {
+    if (!config.webpackConfig) {
+      // eslint-disable-next-line no-console
+      console.log('[Terra-Toolkit:serve-static] No webpack configuration provided');
+      return;
+    }
+
+    const webpackConfig = config.webpackConfig;
+    const port = ((config || {}).serveStatic || {}).port || 8080;
+    const index = ((config || {}).serveStatic || {}).index || 'index.html';
+
+    // If no output is provided, define one.
+    if (!(webpackConfig.output || {}).path) {
+      webpackConfig.output = Object.assign({}, webpackConfig.output, { path: '/dist' });
```

Comment:
should we provide '/dist' as the output path in default webpack config [here](https://github.com/cerner/terra-toolkit/pull/81/files#diff-aafcef2bb82f2b55a3cc6fdd844d14e0R106)?

Reply:
This service does not assume the consumer is using the default webpack config.